### ISSUE TITLE
 Refreshing auth key for channel causes play to stop

### ISSUE THREAD

**Issue by ghost** [interlark/123tv-iptv#8]
Running Raspberry PI OS lite on a RPI3. When tailing journactl, I see an entry of  Refreshing auth key for channel by the 123iptv process. This causes playback to freeze which requires user to stop and then restart the channel.

**Comment by interlark** [comment]
Yeah, that sounds true. It takes some valuable amount of time to refresh all the channels list. USTVGO has only 2 auth keys for VIP and NON-VIP channels, while 123TV force us to update keys for each channel.  I think I could add lazy auth-key refreshing in the next release.
